selfdrive/car/gm/values.py

```diff
@@ -135,6 +138,28 @@ class CanBus:
   LOOPBACK = 128
   DROPPED = 192
 
+FW_QUERY_CONFIG = FwQueryConfig(
+  requests=[
+    Request(
+      [StdQueries.SHORT_TESTER_PRESENT_REQUEST, StdQueries.OBD_VERSION_REQUEST],
+      [StdQueries.SHORT_TESTER_PRESENT_RESPONSE, StdQueries.OBD_VERSION_RESPONSE],
```

Comment:
I have some queries that the dealer uses to get the software part numbers, I just haven't figured out which to use, since one ECU has 10 or so different software codes. Thanks for attempting, but let's remove the FW code for now.

Reply:
Removed and I'll hit you up on Discord.